### ISSUE TITLE
If installer had https then activate https by default

### ISSUE THREAD

**Issue by naox** [PrestaShop/PrestaShop#19991]
If prestashop installer was run through https url then activate https for installed shop by default. This would solve infinite loop on hostings where redirect to https is on by default.

**Comment by Progi1984** [comment]
@naox Hi, a new option has been added in `develop` branch for choosing HTTPS or not during the install : #19791

**Comment by naox** [comment]
Thats good but I strongly recomend setting it initial state based on that if installer is running through https url.

**Comment by Progi1984** [comment]
@naox If I understand, you would like that option would be selected on "Yes" for "Enable SSL" only if the installer is in HTTPS. Right ?

**Comment by naox** [comment]
Installer should should set default value for this option based on if instlaler is running through https.

> you would like that option would be selected on "Yes" for "Enable SSL" only if the installer is in HTTPS

Remove word "only" and then yes.

**Comment by ghost** [comment]
Hey @Progi1984 i think this issue is fixed but i don't find the PR

**Comment by Progi1984** [comment]
@okom3pom No, the installer has not a default value. We need to wait the answer of @PrestaShop/product-team on this feature.
